packages/node/src/node.ts

```diff
@@ -1,35 +1,58 @@
+import { ethers } from "ethers";
 import EventEmitter from "eventemitter3";
+import firebase from "firebase";
 
 import {
+  Address,
   AppInstanceInfo,
   GetAppInstancesResult,
   MethodName,
   MethodRequest,
   MethodResponse
 } from "./node-types";
 
+const MESSAGES = "messages";
+
 export default class Node {
+  /**
+   * The event that Node consumers can listen on for messages from other
+   * nodes.
+   */
+  public static PEER_MESSAGE = "peerMessage";
+
   /**
    * Because the Node receives and sends out messages based on Event type
    * https://github.com/counterfactual/monorepo/blob/master/packages/cf.js/src/types/node-protocol.ts#L21-L33
    * the same EventEmitter can't be used since response messages would get
    * sent to listeners expecting request messages.
    **/
-  private incoming: EventEmitter;
-  private outgoing: EventEmitter;
+  private readonly incoming: EventEmitter;
+  private readonly outgoing: EventEmitter;
+  private readonly signer: ethers.utils.SigningKey;
```

Comment:
You're currently not signing anything with this key, only using its public address. Is that going to change in the near future?

Reply:
Yes. Embedding the machine will effectively make more use of this.